yt_dlp/extractor/minitv.py

```diff
@@ -0,0 +1,337 @@
+import json
+from copy import copy
+
+from .common import InfoExtractor
+from ..utils import ExtractorError, int_or_none, traverse_obj, try_get
+
+
+class MiniTVIE(InfoExtractor):
+    _VALID_URL = r'(?:https?://(?:www\.)?amazon\.in/minitv/tp/|minitv:(?:amzn1\.dv\.gti\.)?)(?P<id>[a-f0-9-]+)'
```

Comment:
why the keyword?

Reply:
Just for consistency with `minitv:season` and `minitv:series`.

Reply:
ig it's fine